### PR TITLE
Adding ASP.NET 5 topic links

### PR DESCRIPTION
**Pull request by danroth27**
#860

### CODE REVIEW

File: aspnet/fundamentals/application-insights.rst [hunk 1]
```diff
@@ -121,4 +121,5 @@ Additional Resources
 --------------------
 
 - `Application Insights API for custom events and metrics <http://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/>`_
-- `Application Insights for ASP.NET 5 <http://blogs.msdn.com/b/webdev/archive/2015/05/19/application-insights-for-asp-net-5-you-re-in-control.aspx>`_
+- `Application Insights for ASP.NET 5 <https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/>`_
+- `Application Insights - Blog post <http://blogs.msdn.com/b/webdev/archive/2015/05/19/application-insights-for-asp-net-5-you-re-in-control.aspx>`_
```
Comment (shirhatti) [inline aspnet/fundamentals/application-insights.rst]: http -> https

**Comment by danroth27** [comment]
Additional changes:
- Removed duplicate content
- Organized Azure specific content in separate sections
- Added links to Azure AD topics

**Comment by danroth27** [comment]
@shirhatti @rachelappel Please review

**Comment by shirhatti** [comment]
As a consequence of this change, customers lose a sample app using AppInsights. I couldn't find a sample app linked to from [here](https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/)

Besides that it looks good to me.

**Pull request merged**